ngrinder-controller/src/main/java/org/ngrinder/starter/NGrinderControllerStarter.java

```diff
@@ -243,6 +248,23 @@ public static void main(String[] args) {
 		SpringApplication.run(NGrinderControllerStarter.class, args);
 	}
 
+	private static void checkTmpDirProperty() {
+		String javaTmpDir = System.getProperty("java.io.tmpdir");
+		String systemTmpDir = System.getenv("TMPDIR");
+		if (javaTmpDir.equals(systemTmpDir)) {
+			System.out.print(CONSOLE_COLOR_YELLOW);
+			System.out.println("####################################################################################################################");
+			System.out.println("# WARNING                                                                                                          #");
+			System.out.println("# If you run the ngrinder-controller as SpringBoot JAR without specifying 'java.io.tmpdir' system property,        #");
+			System.out.println("# related library files could be unintentionally missing and you could encounter unexpected errors.                #");
+			System.out.println("# You must run the ngrinder-controller with specifying 'java.io.tmpdir' system property.                           #");
+			System.out.println("#   ex) java -Djava.io.tmpdir=/home/user/.ngrinder/lib -jar ngrinder-controller.war                                #");
+			System.out.println("####################################################################################################################");
+			System.out.print(CONSOLE_COLOR_RESET);
+			System.exit(1);
```

Comment:
too complex message.
please fix like "Wrong runtime option. please put tmpdir property like following. ```java -Djava.io.tmpdir=/home/user/.ngrinder/lib -jar ngrinder-controller.war```

Reply:
please make this message shown as the first message...

Reply:
Changed it. Message shows like this.
![image](https://user-images.githubusercontent.com/10591327/209750137-59236502-2b2c-4901-9f81-6b0062e28117.png)

Reply:
What is proper exit code? -1 ? 1